frontend/amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -43,6 +43,8 @@ import { logAction } from 'utils/analytics';
 import { buildTableKey, TablePageParams } from 'utils/navigationUtils';
 import { getUniqueValues, filterOutUniqueValues } from 'utils/stats';
 
+import { GraphIcon } from 'components/SVGIcons/GraphIcon';
```

Comment:
Could you add a couple of extra test cases in 'js/features/ColumnList/index.spec.tsx', in the 'when columns with one usage data entry are passed' and 'when columns with no usage data are passed', looking for this icon to show up?